libmscore/page.cpp

```diff
@@ -399,9 +407,11 @@ void PageFormat::read(XmlReader& e, Score* score)
                         }
                   }
             else if (tag == "page-height")
-                  _size.rheight() = e.readDouble() * 0.5 / PPI;
+                  _size.rheight() = e.readDouble() / SCALE_XML;
             else if (tag == "page-width")
-                  _size.rwidth() = e.readDouble() * .5 / PPI;
+                  _size.rwidth() = e.readDouble() / SCALE_XML;
+            else if (tag == "page-units")
```

Comment:
The file format is only changing via the addition of the "page-units" tag.  I believe the question here is about the contents of the "page-units" tag being an integer, Units, or a string, unitSuffixes[].  Do we store the number one, or the string "mm", for example?
The units used in the file storage for page settings values have not changed.  They have always been at 144dpi.